packages/ihe-gateway/server/CodeTemplates/Common Library/Check if document exists in AWS S3 Bucket .xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?><codeTemplate version="4.4.2">
+  <id>e55a3d2d-bafe-4225-8194-b2a876e6e67a</id>
+  <name>Check if document exists in AWS S3 Bucket </name>
```

Comment:
Can we please use a name that matches our pattern for file names, knowing now that this will be become a file on our repository?

Reply:
Oh i wasnt aware, thanks for the heads up